packages/teams-js/src/public/stageView.ts

```diff
@@ -115,4 +115,39 @@ export namespace stageView {
   export function isSupported(): boolean {
     return ensureInitialized(runtime) && runtime.supports.stageView ? true : false;
   }
+
+  export namespace stageClose {
+    /**
+     * Closes the current stage view.
+     * @returns Promise that resolves or rejects with an error once the stage view is closed.
+     */
+    export function closeCurrentWindow(): Promise<void> {
```

Comment:
This function should have e2e tests added for it. It currently has platform-agnostic e2e tests defined in `stageView.json`, but you will probably need to add a new e2e test file since this is a new capability, something like `stageView.self.json`. I will send you a link to documentation on how to add these, along with where to ask e2e test-specific questions